src/constants.js

```diff
@@ -27,7 +27,7 @@ const substrateNetworkBase = {
     decimals: 12,
     genesisHash: SubstrateNetworkKeys.KUSAMA,
     prefix: 2,
-    title: 'Kusama',
+    title: 'Kusama CC2',
```

Comment:
>  Anything you’ve done on Kusama so far will persist into Kusama CC2
See this from the CC2 release post here, correct me if I am wrong, we can still use name `Kusama`, since currently all the users on Kusama use the same network.

Reply:
We can still use the name Kusama, but if there's a CC3 in the future, with a different genesis hash, and we only write "Kusama", things may be confusing, I think it's better to be precise.

If there's a Kusama CC3, any account created with CC2 will need to be migrated (manually or automatically).

Reply:
Agree on the naming.